README.md

```diff
@@ -172,3 +172,14 @@ Functional tests are written with PHPUnit. Note that Bundles and Components are
 ## Remove demo configuration
 
 If you start a project from the sandbox, remove ```.sensiolabs.yml``` as its not a good example for production use.
+
+## Apply vagrant box
+
+There's a new vagrant box that is simple to boot:
```

Comment:
the readme is not time-based so we should not say "new vagrant box" but simply: "The easiest way to set up the sandbox is using Vagrant:"